packages/common/src/intl/locales/en/common.json

```diff
@@ -127,5 +127,7 @@
   "link.history": "History",
   "placeholder.enter-a-customers-name": "Enter a customers name",
   "placeholder.enter-a-suppliers-name": "Enter a suppliers name",
-  "placeholder.enter-an-item-code-or-name": "Enter item code or name"
-}
+  "placeholder.enter-an-item-code-or-name": "Enter item code or name",
+  "label.counted-num-of-packs": "Counted # of Packs",
+  "label.snapshot-num-of-packs": "Snapshot # of Packs"
+}
```

Comment:
I've forgotten to do the translation issue to make the data table name space before this so have just doubled up a bit for now